_posts/2017-01-09-belfast-open-data.md

```diff
@@ -0,0 +1,15 @@
+---
+layout: post
+title: Designing Real Time Data Applications - Talk at the Detail Data Conference
+date: 2017-01-09 10:00
+author: jase
```

Comment:
change author to jason as per https://github.com/MastodonC/mastodonc.github.com/pull/25/files